tensorflow_datasets/url_checksums/breastpathq.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
The checksum needs to be added, additionally since the competition gives a zip. You can still extract the zip instead of making the user do so.

Reply:
https://colab.research.google.com/drive/1tIzoarbInwB7jduej1u8ACNLRNT_7v1e

Here I must manually unzip, please use the extract function of `dl_manager` to unzip once the data is downloaded.